smart_proxy_remote_execution_ssh.gemspec

```diff
@@ -29,7 +29,6 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency('rack-test', '~> 0')
   gem.add_development_dependency('rubocop', '~> 0.82.0')
 
-  gem.add_runtime_dependency('foreman-tasks-core', '>= 0.3.1')
-  gem.add_runtime_dependency('smart_proxy_dynflow', '~> 0.1')
+  gem.add_runtime_dependency('smart_proxy_dynflow', '~> 0.5')
```

Comment:
I'm confused with versioning, is the current 0.4 sufficient, or are we releasing 0.5 for that?

Reply:
0.4 is sp dynflow core to sp dynflow, 0.5 will be tasks core -> sp dynflow. Getting 0.4 through packaging is taking a bit longer than I expected, I'd like to phase out the releases a bit so we don't drop a bunch of releases changing everything at the same time